.github/workflows/ci.yml

```
@@ -0,0 +1,83 @@
+          --health-timeout 3s
+          --health-retries 10
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Wait for Valkey
+        run: |
+          for i in {1..30}; do
+            if python3 - <<'PY'
+import socket
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 33: could not find expected ':'

(syntax-check)